readme.md

```diff
@@ -1585,6 +1585,44 @@ const {clear} = render(<MyApp />);
 clear();
 ```
 
+#### measureElement(ref)
+
+Measure the dimensions of a particular `<Box>` element.
+It returns an object with `width` and `height` properties.
+This function is useful when your component needs to know the amount of available space it has before it can start rendering or when you need to change your layout based on the length of the content.
```

Comment:
This line sounds a bit confusing to me, especially the "before it can start rendering part".

My suggestion would be:

"This function is useful when your component needs to know the amount of available space it has. You could use it, for example, when you need to change layout based on the length of its content."